test/unit/adb-commands-specs.js

```diff
@@ -576,6 +576,26 @@ describe('adb commands', () => {
       });
     }));
   });
+  describe('device info', withMocks({adb}, (mocks) => {
+    it('should get device model', async () => {
+      mocks.adb.expects("shell")
+          .once().withArgs('getprop');
+      await adb.getModel();
```

Comment:
For all of these, the mock ought to return expected results, so the method's logic is tested.